### PR TITLE
Add new files

### PR DESCRIPTION
**Pull request by PythonF4DE**
I added file for 5th lab and added more description for repo

### CODE REVIEW

**Review by AyToFix — APPROVED** [review]
All is great, i guess(Commented by Bohdan Psiuk)

**Comment by PythonF4DE** [comment]
> All is great, i guess(Commented by Bohdan Psiuk)

One Yenot for

**Comment by PythonF4DE** [comment]
> All is great, i guess(Commented by Bohdan Psiuk)

You get one racoon from English because of lowercase "I" when it is supposed to be capital letter

File: lab5.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+"""Code for lab work 5 from programming course"""
+class Fish:
+    """This class creates a simple object fish that has some parameters"""
+    def __init__(self, name, size, is_agressive, needed_space):
+        self.name = name
+        self.size = size
+        self.is_agressive = is_agressive
+        self.needed_space = needed_space
```
Comment (PythonWizardry) [inline lab5.py:8]: Docstring for this Class is not needed

File: lab5.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        self.is_agressive = is_agressive
+        self.needed_space = needed_space
+
+
+class Aquarium:
+    """This class creates an object called aquarium for putting fish in it"""
+    def __init__(self, total_volume):
+        self.total_volume = total_volume
+        self.free_space = total_volume
+        self.list_of_fish = []
```
Comment (PythonWizardry) [inline lab5.py:16]: I think for this Class Docstring is not needed either

**Review by PythonWizardry — COMMENTED** [review]
Code is very good, you should just delete some of Docstring

**Pull request closed without merging**